docs/upgrade_with_cli.md

```diff
@@ -0,0 +1,51 @@
+# Upgrade Percona Everest using the CLI
+
+Percona Everest regularly releases updates that contain bug fixes, security improvements, and other enhancements to improve the overall performance of your database.
+
+
+## Before you upgrade
+
+!!! note
+     Ensure that you have installed the latest version of the CLI.
+
+Upgrades to Percona Everest are supported from version 0.9.0 onwards. However, you can only upgrade to one version at a time. For instance, you can upgrade from version 0.9.1 to version 0.10.0, but not directly from version 0.8 to version 0.10.  
```

Comment:
We should say that users need to download CLI version >=0.10.0 for the upgrade command to work.